src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -16,6 +16,7 @@ public class MovieDaoImpl implements MovieDao {
     public Movie add(Movie movie) {
         Transaction transaction = null;
         Session session = null;
```

Comment:
Consider using a try-with-resources statement for the `Session` object to ensure it is closed properly, even if an exception occurs. This would align with the pattern used in the `get` and `getAll` methods.